vizro-core/tests/unit/vizro/models/_action/test_action.py

```diff
@@ -150,7 +150,7 @@ def managers_one_page_without_graphs_one_button():
     vm.Page(
         id="test_page",
         title="Test page",
-        components=[vm.Graph(figure=px.scatter(data_frame=pd.DataFrame(columns=["A"]), x="A", y="A"))],
+        components=[vm.Graph(figure=px.scatter(data_frame=pd.DataFrame(data={"A": [1], "B": [2]}), x="A", y="B"))],
```

Comment:
This has changed because an empty dataframe with a filter now eagerly raises an error message, which it wouldn't have done before (I think something else probably would have broken when viewing the dashboard before, just we didn't raise an error message in advance).